nodePackages/web/src/components/molecules/ChangeThemeIcon.tsx

```
@@ -0,0 +1,67 @@
+  const concProps = mergeProps(
+    {
+      dark: 'dark',
+      labelToDark: '',
+      labelToLight: '',
+      light: 'light',
+    } as const,
+    props,
+  );
+  const isDarkMode = createDarkMode(concProps);
+  let ref!: HTMLInputElement;
+  onMount(() => (ref.checked = isDarkMode()));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)